Framework/DataObjects/test/EventListTest.h

```diff
@@ -55,6 +57,28 @@ class EventListTest : public CxxTest::TestSuite {
     el = EventList(mylist);
   }
 
+  void test_copyDataFrom() {
```

Comment:
Minor point. See to be testing both incompatibility of types and correctly copy behaviour in one test. Could split into two.

Reply:
Another minor point. Is it worth testing other aspects that are supposed to be copied? Ordering, Weighted Events etc?